pyo3-macros-backend/src/pyclass.rs

```diff
@@ -379,12 +390,21 @@ struct PyClassEnumVariant<'a> {
 }
 
 impl<'a> PyClassEnumVariant<'a> {
-    fn python_name(&self) -> Cow<'_, syn::Ident> {
-        self.options
+    fn python_name(&self, args: &PyClassArgs) -> Cow<'_, syn::Ident> {
+        let name = self
+            .options
             .name
             .as_ref()
             .map(|name_attr| Cow::Borrowed(&name_attr.value.0))
-            .unwrap_or_else(|| Cow::Owned(self.ident.unraw()))
+            .unwrap_or_else(|| Cow::Owned(self.ident.unraw()));
+        if args.options.rename_all.is_some() {
+            Cow::Owned(Ident::new(
+                &format!("{}", name).to_shouty_snake_case(),
```

Comment:
Considering [the analogy with serde](https://serde.rs/container-attrs.html#rename_all), I would have expected to specify the convention as an argument to `rename_all`.

I understand that this mainly targets making enum variants matching the Python conventions, but I wonder whether we should give this a more specific name then, e.g. `rename_variants`?

Reply:
`rename_all` now behaves exactly like serde's one. Keeping the name as is.